src/format.ts

```diff
@@ -35,6 +35,7 @@ enum Design {
 	PrintShop,
 	Obituary,
 	Correction,
+	FullWidthInteractive,
```

Comment:
I'd suggest maybe `FullPageInteractive` since this actually now refers to the existing type of interactives that take over the whole page?

Reply:
"Full page" makes me think the nav, header, footer etc is being taken over - which isn't the case on DCR

Reply:
I do agree here that 'FullWidth' is a bit too specific. It makes me think of `width: 100%` but I think this `Design` means more than this?

Reply:
> "Full page" makes me think the nav, header, footer etc is being taken over - which isn't the case on DCR

The header _is_ actually being taken over, as this `Design` will apply to the existing interactives that take over everything except the thin nav and the footer. I don't know if there's a better word for that though (naming things is hard).